derive/src/lib.rs

```diff
@@ -263,6 +272,15 @@ impl Parse for Parameter {
 
                 Parameter::Methods(methods)
             }
+            "event_derives" => {
+                let content;
+                parenthesized!(content in input);
+                let derives = content
+                    .parse_terminated::<_, Token![,]>(TypePath::parse)?
```

Comment:
This should be a `Path` and not a `TypePath`, type paths allow for `<Foo as Bar>::Type` which is not allowed for `derive`s AFAIK.